README.md

```diff
@@ -1 +1,29 @@
-# subspace-cli
\ No newline at end of file
+# Subspace-CLI
+is a great tool that heavily simplifies the terminal workflow of the farming process. \\
+Instead of running a terminal instance for farmer, and running another terminal instance for the node, now you can run a SINGLE terminal instance to farm!
+
+## How to Use (commands)
+1. download the executable
+2. in your terminal, change your directory to where you download the file for example: if you downloaded your file to your `Downloads` folder, `cd Downloads`)
+3. run `./subspace-cli init` -> this will initialize your config file, which will store the necessary information for you to farm.
+4. run `.subspace-cli farm` -> this will start farming. Yes, it is that simple! Enjoy!
```

Comment:
```suggestion
3. run `subspace init` -> this will initialize your config file, which will store the necessary information for you to farm.
4. run `subspace farm` -> this will start farming. Yes, it is that simple! Enjoy!
```

Reply:
Maybe it would be better to strip `-cli` from the name, as it means nothing

Reply:
Yeah, makes sense related to `./`

Reply:
let me fix the typo though